tests/conftest.py

```diff
@@ -39,4 +36,64 @@ def get_neural_network_data(desc):
         x_test = np.random.uniform([-1, 2], [1, 3], (5, 2))
         return x, y, x_test
 
-    return None
\ No newline at end of file
+    return None
+
+
+class _Datadir:
+    """
+    Give access to files in the `models` directory.
+    """
+
+    def __init__(self, basedir):
+        self._basedir = basedir
+
+    def file(self, filename):
+        return str(self._basedir / filename)
+
+
+@pytest.fixture
+def datadir():
+    basedir = Path(this_file_dir()) / "models"
+    return _Datadir(basedir)
+
+
+@pytest.fixture
+def two_node_network_relu():
+    """
+            1           1
+    x0 -------- (1) --------- (3)
+     |                   /
+     |                  /
+     |                 / 5
+     |                /
+     |               |
+     |    -1         |     1
+     ---------- (2) --------- (4)
+    """
+    net = NetworkDefinition(scaled_input_bounds=[(-10.0, 10.0)])
+
```

Comment:
technically `scaled_input_bounds` should be a dict, but this works because we get lucky and the key from pyomo is just `[0]`.

Reply:
I changed this.